sync_issues_to_jira/sync_issue.py

```diff
@@ -170,6 +170,8 @@ def _markdown2wiki(markdown):
     """
     Convert markdown to JIRA wiki format. Uses https://github.com/Shogobg/markdown2confluence
     """
+    if markdown is None:
+        markdown = ""  # Allow empty/blank input
```

Comment:
Minor request, can we simply return "\n" here without running markdown2confluence?

Reply:
Nice catch! Thanks, updated